react/tsconfig.json

```diff
@@ -30,5 +30,5 @@
     "pretty": true,
     "sourceMap": true
   },
-  "include": ["src"]
+  "include": ["src", "../statviz/src/views/Statistics", "../statviz/src/components/graphs"]
```

Comment:
@MaikNeubert please remove the latter two paths. They are the reason for the build failure. statviz has its own tsconfig and this way they are intersecting where they should not.